pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -112,6 +112,7 @@ type KogitoServiceSpecInterface interface {
 	GetRuntime() RuntimeType
 	//Define port on which service will listen internally
 	GetHTTPPort() int32
+	IsInsecureImageRegistry() bool
```

Comment:
If we add this flag here, the `KogitoRuntime` will also inherit it. Is that right?

Reply:
Yes